src/RuleSet/Sets/PERCSSet.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\RuleSet\Sets;
+
+use PhpCsFixer\RuleSet\AbstractRuleSetDescription;
+
+/**
+ * @internal
+ */
+final class PERCSSet extends AbstractRuleSetDescription
+{
+    public function getName(): string
```

Comment:
I suggest PHP CS Fixer set to use newest PER-CS and not PER-CS2.0

Reply:
I thought about it, but decided to not mix separate things (introducing new set vs changing existing one). But I can change it if you want 🙂.

Reply:
please do (same or new PR). thanks!